crowbar_framework/lib/crowbar/logger/formatter.rb

```diff
@@ -18,20 +18,40 @@ module Crowbar
   module Logger
     class Formatter < ::Logger::Formatter
       include ActiveSupport::TaggedLogging::Formatter
+      Format = "%s, [%s#%d] %5s -- %s\n"
+      ThreadFormat = "%s, [%s#%d:%s] %5s -- %s\n"
```

Comment:
Style/ConstantName: Use SCREAMING_SNAKE_CASE for constants. (https://github.com/bbatsov/ruby-style-guide#screaming-snake-case)<br>Style/MutableConstant: Freeze mutable objects assigned to constants.